### PR TITLE
Add more tests for nonstandard orders

### PR DESCRIPTION
**Pull request by zhongeric**

### CODE REVIEW

File: test/foundry-tests/reactors/RelayOrderReactorTest.t.sol [hunk 1]
```diff
@@ -176,6 +192,21 @@ contract RelayOrderReactorTest is GasSnapshot, Test, PermitSignature, DeployPerm
         reactor.execute(signedOrder, filler);
     }
 
+    function test_execute_noInputs_reverts_SignatureExpired() public {
```
Comment (snreynolds) [inline test/foundry-tests/reactors/RelayOrderReactorTest.t.sol:195]: nit: can you name this test_execute_reverts ? its nice for regex matching and for consistency

File: test/foundry-tests/reactors/RelayOrderReactorTest.t.sol [hunk 2]
```diff
@@ -176,6 +192,21 @@ contract RelayOrderReactorTest is GasSnapshot, Test, PermitSignature, DeployPerm
...
+        uint256 deadline = block.timestamp + 10;
+        Input[] memory inputs = new Input[](0);
+        RelayOrder memory order = RelayOrderBuilder.initDefault(tokenIn, address(reactor), swapper);
```
Comment (snreynolds) [inline test/foundry-tests/reactors/RelayOrderReactorTest.t.sol:198]:
```suggestion
        Input[] memory inputs = new Input[](0);
        RelayOrder memory order = RelayOrderBuilder.initDefault(tokenIn, address(reactor), swapper);
        order.inputs = inputs;
       
```

File: test/foundry-tests/reactors/RelayOrderReactorTest.t.sol [hunk 3]
```diff
@@ -176,6 +192,21 @@ contract RelayOrderReactorTest is GasSnapshot, Test, PermitSignature, DeployPerm
...
+        order.info = order.info.withDeadline(deadline);
+
+        SignedOrder memory signedOrder =
+            SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
+
+        vm.warp(deadline + 1);
+        vm.prank(filler);
```
Comment (snreynolds) [inline test/foundry-tests/reactors/RelayOrderReactorTest.t.sol:205]: just a note but its not necessary to prank

**Review by snreynolds — CHANGES_REQUESTED** [review]

File: test/foundry-tests/reactors/RelayOrderReactorTest.t.sol (hunk 2 above)
Comment (snreynolds) [reply test/foundry-tests/reactors/RelayOrderReactorTest.t.sol:198]: initDefault initializes with 1 input

Comment (zhongeric) [reply test/foundry-tests/reactors/RelayOrderReactorTest.t.sol:198]: ah nice yea

**Comment by snreynolds** [comment]
> Forge code coverage:
> 
> File	% Lines	% Statements	% Branches	% Funcs
> src/base/Multicall.sol	100.00% (6/6)	100.00% (8/8)	100.00% (2/2)	100.00% (1/1)
> src/lens/RelayOrderQuoter.sol	100.00% (18/18)	100.00% (24/24)	100.00% (8/8)	100.00% (7/7)
> src/lib/ActionsLib.sol	0.00% (0/4)	0.00% (0/7)	0.00% (0/2)	0.00% (0/1)
> src/lib/InputsLib.sol	0.00% (0/15)	0.00% (0/22)	100.00% (0/0)	0.00% (0/3)
> src/lib/RelayDecayLib.sol	0.00% (0/11)	0.00% (0/13)	0.00% (0/8)	0.00% (0/1)
> src/lib/RelayOrderLib.sol	0.00% (0/22)	0.00% (0/34)	0.00% (0/6)	0.00% (0/3)
> src/reactors/RelayOrderReactor.sol	100.00% (9/9)	100.00% (11/11)	100.00% (0/0)	100.00% (2/2)
> Total	26.36% (34/129)	24.58% (44/179)	38.46% (10/26)	27.50% (11/40)

omg look at her goooo

**Review by snreynolds — APPROVED** [review]

**Pull request merged**